src/Controller/Command/CommandListController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Controller\Command;
+
+use App\Entity\{
+    Command,
+    Menu,
+    Product,
+    Category
+};
+
+use App\ORM\EntityManager;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class CommandListController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        $em = new EntityManager();
```

Comment:
oh you should have used the one in the container to reduce the memory consumption and share the same object over the application life.